src/Features/CSharp/Portable/xlf/CSharpFeaturesResources.cs.xlf

```diff
@@ -157,11 +157,6 @@
         <target state="translated">Zjednodušit všechny výskyty</target>
         <note />
       </trans-unit>
-      <trans-unit id="Remove_Unnecessary_Usings">
-        <source>Remove Unnecessary Usings</source>
-        <target state="translated">Odebrat nepotřebné direktivy using</target>
```

Comment:
💡 It would have been nice to preserve these strings. Since the target file is empty, it can be done by copying all of these XLF files over the ones in the new location, renaming them, and running a build. All the unnecessary strings will be removed, leaving translated moved strings.

It could also be done with a simple C# script using System.Xml.Linq and XPath.

Reply:
I presume this is only a one time issue when the target file is empty? Subsequent resource moves should be preserved by git history correct?